config/settings.yml

```diff
@@ -0,0 +1,6 @@
+links:
+  rails_tutorial:
+    home_page: http://www.railstutorial.org/
+    help_page: http://www.railstutorial.org/help
+    help_book: http://www.railstutorial.org/book
+    
```

Comment:
add end line (blank line) for this file

Reply:
![image](https://user-images.githubusercontent.com/68435978/90864476-53aa0280-e3c3-11ea-868f-e805e9bd9973.png)
I have add end line already.